React_Day4/avengers/src/App.css

```diff
@@ -0,0 +1,28 @@
+body{
+    background-color: black;
+}
+
+#card{
+    width: 250px;
+    border : 2px solid brown;
+    padding: 10px;
+    margin: 15px;
+}
+
+#card > img{
+    width: 100%;
+}
+
+#applayout{
+    display: flex;
+}
+
+h1{

```

Comment:
Use html selectors, *, body up above the sky. Always is good when you follow CSS specificity.